src/app/product.component.scss

```diff
@@ -0,0 +1,27 @@
+.FormSearch {
+    min-width: 150px;
+    max-width: 500px;
+    width: 100%;
+  }
+  
+  .FormField {
+    width: 100%;
+  }
+  .Card {
+    max-width: 400px;
```

Comment:
name in CSS does not start with capital letters as class-name

Reply:
https://www.tutorialspoint.com/which-characters-are-valid-in-css-class-names-selectors